pkg/logql/log/fmt_test.go

```diff
@@ -511,6 +511,26 @@ func mustNewLabelsFormatter(fmts []LabelFmt) *LabelsFormatter {
 	return lf
 }
 
+// Test_Panic checks if a custom function that use regular expressions panics.
+// This panic is caught inside the text/template library when the function is called.
+func Test_Panic(t *testing.T) {
```

Comment:
Why do we want it to panic? Can't we validate the regex and return an error message?

Reply:
Yes indeed, that also works.